src/main/java/com/google/sps/servlets/EventServlet.java

```diff
@@ -251,6 +252,7 @@ private String addEvent(HttpServletRequest request, HttpServletResponse response
     String eventId = null;
     try {
       eventId = eventStorageObject.addOrUpdateEvent(event);
+      userStorageObject.joinEvent(currentUserId, eventId, true);
```

Comment:
Optional: For better readability you can emphasize the param name when you use literals like `true`, e.g. here:
`userStorageObject.joinEvent(currentUserId, eventId, /*paramName=*/true);`